humility-core/src/hubris.rs

```diff
@@ -4546,7 +4547,11 @@ impl HubrisArchive {
         //
         let mut written = 0;
 
-        let started = Instant::now();
+        let started = match started {
+            Some(s) => s,
+            None => Instant::now(),
+        };
```

Comment:
```suggestion
        let started = started.unwrap_or_else(Instant::now);
```

Reply:
Yeah, really need to get my fingers better at typing this one. Fixed!